### DISCUSSION TITLE
Unroute Path on Apple Silicon Mac

### DISCUSSION THREAD

**Discussion by joshuaoliverio** [Homebrew/discussions#3446] [Getting started]
When setting up it says:

==> Next steps:
- Add Homebrew to your PATH in /Users/joshuaoliverio/.zprofile:
    echo 'eval "$(/opt/homebrew/bin/brew shellenv)"' >> /Users/joshuaoliverio/.zprofile
    eval "$(/opt/homebrew/bin/brew shellenv)"

How do I undo this?

**Comment by carlocab** [comment]
```
open ~/.zprofile
```
Remove the line that says
```
eval "$(/opt/homebrew/bin/brew shellenv)"
```

**Reply by ZohaibAhmad786** [reply]
thanks bro
